GTViewController/Classes/Nav Classes/GTFollowupViewController.m

```diff
@@ -296,4 +270,55 @@ - (void)alertView:(UIAlertView *)alertView clickedButtonAtIndex:(NSInteger)butto
     [alertView dismissWithClickedButtonIndex:buttonIndex animated:YES];
 }
 
+
+#pragma mark - UISnuffleButtonTapDelegate methods
+
+- (void)didReceiveTapOnPositiveButton:(UISnuffleButton *)positiveButton {    
+    NSArray *inputValidationErrors = [self inputValidationErrors];
+    
+    if ([positiveButton forceValidation] && [inputValidationErrors count]) {
```

Comment:
You are right that it should be rechecked using the current pattern. If the pattern is changed you would not get the button as a parameter and it would not get checked again.